hummingbot/strategy/arbitrage/arbitrage.pyx

```diff
@@ -582,8 +582,10 @@ cdef class ArbitrageStrategy(StrategyBase):
                                     f"Base asset needed: {total_bid_value + bid_price * amount}. "
                                     f"Base asset balance: {sell_market_base_asset}. ")
 
+                # max market buys need to be reduced to account for additional fees
+                adjusted_buy_market_quote_asset = (buy_market_quote_asset - total_buy_flat_fees) / (1 + buy_fee.percent)
```

Comment:
rename adjusted_buy_market_quote_asset to make it more clear. It should indicate it is the order amount you are placing.

Reply:
you need to convert quote to base amount